jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/newcase/NewCaseInstanceViewImpl.java

```diff
@@ -122,14 +155,19 @@ public HTMLElement getElement() {
     }
 
     private void createCaseInstance() {
-        presenter.createCaseInstance(caseTemplatesList.getValue());
+        presenter.createCaseInstance(caseTemplatesList.getValue(), ownerNameInput.getValue());
     }
 
     private void clearErrorMessages() {
         definitionNameHelp.setTextContent("");
```

Comment:
@cristianonicolai I think we have to clean here previous error messages related with the owner